src/main/java/com/faforever/client/rankedmatch/Ladder1v1Controller.java

```diff
@@ -155,6 +167,39 @@ public void initialize() {
 
     JavaFxUtil.addListener(playerService.currentPlayerProperty(), (observable, oldValue, newValue) -> Platform.runLater(() -> setCurrentPlayer(newValue)));
     playerService.getCurrentPlayer().ifPresent(this::setCurrentPlayer);
+    
+    gameService.addOnRankedMatchNotificationListener(message -> {
+      if (message.getQueues() == null) {
+        return;
+      }
+      for (MatchmakerMessage.MatchmakerQueue matchmakerQueue : message.getQueues()) {
+        if (!Objects.equals("ladder1v1", matchmakerQueue.getQueueName())) {
```

Comment:
put "ladder1v1" into a constant

Reply:
Ok  it think MainController uses this as well, will replace it there aswell.

Reply:
I think i can make this an enum, GSON can apparently deserialize straight into an enum. Gonna put it into MatchmakerQueue.java i think.